db/test/test_stats.py

```diff
@@ -46,7 +46,7 @@ def test_get_last_submitted_recordings(self, dbset, dbget):
                     "version": {"essentia_build_sha": "sha"},
                 },
             }
-            db.data.write_low_level(rand_mbid, data, gid_types.GID_TYPE_MBID)
+            db.data.write_low_level(str(rand_mbid), data, gid_types.GID_TYPE_MBID)
```

Comment:
did this stop working without the call to `str()`? I'm interested to know if we should make sure that other places that call this still work - do all places that call this method in the main code pass a string, or a uuid object?

Reply:
This did stop working without the call to str(), and I believe that it was because I casted the gid to text in the query for submission offset. This was the error: 
<img width="1679" alt="Screen Shot 2019-05-18 at 12 00 20 PM" src="https://user-images.githubusercontent.com/30450017/57972707-18bd5180-796c-11e9-93a8-8864d73d687f.png">

I removed the cast to text in the query and removing the call to `str()` allowed the test to pass. The api endpoint for `submit_low_level` makes a call to `str()` on the mbid, but in `db/data.py` the function `submit_low_level_data` also casts the mbid as a string before passing it to `write_low_level`. 

So typically, the main code passes a string rather than a uuid. But it appears to me that the comparison between gid in the lowlevel table and a string mbid works without using `gid::text`. So by removing gid::text, we can then use write_low_level with both strings and uuids?